src/common/Color.h

```diff
@@ -35,7 +35,6 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <type_traits>
 
 #include "Compiler.h"
-#include "Math.h"
```

Comment:
It seems that this removal is making the appveyor fail.

Reply:
Looks like line 272 uses math stuff. No idea why it works with GCC.